### ISSUE TITLE
The weight of the loss function

### ISSUE THREAD

**Issue by CharlesYTJ** [onpix/LCDPNet#9]
Hello, I am very enlightened after reading this paper. However, the paper mentions in the loss function section that four hyperparameters are selected in order to balance the four loss functions, and there are more details in the supplementary materials. Then can you provide supplementary materials? I would like to know if these four hyperparameters are set based on experience or for some other reason. I look forward to your answer. Thank you.

**Comment by onpix** [comment]
The loss weights are as shown in the codebase. i.e., the weights of the reconstruction loss, cosine loss, and two smoothness losses are 1.0, 0.5, 0.01, 0.01, respectively.

**Comment by CharlesYTJ** [comment]
> 

Thank you for your answer.
